src/app/shared/components/page-not-found/page-not-found.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-page-not-found',
+  templateUrl: './page-not-found.component.html',
+  styleUrls: ['./page-not-found.component.scss'],
+})
+export class PageNotFoundComponent {
+  constructor() {}
```

Comment:
Delete `constructor` if you don't use it.

Reply:
remove